schema.graphql

```diff
@@ -42,59 +88,119 @@ enum Chain {
 }
 
 type RootGauge @entity {
+  " RootGauge contract address"
   id: ID!
+
+  " Chain where funds should be sent "
   chain: Chain!
+
+  " Address of the contract (from the chain above) that receives the BAL emissions "
   recipient: Bytes!
+
+  " Whether Balancer DAO killed the gauge "
   isKilled: Boolean!
+
+  " Relative weight cap for the gauge "
   relativeWeightCap: BigDecimal
 }
 
 type Gauge @entity {
+  " Equal to: <gaugeAddress>-<typeID> "
   id: ID!
+
+  " Address of the gauge "
   address: Bytes!
+
+  " Type of the gauge "
   type: GaugeType!
+
+  " Reference to LiquidityGauge "
   liquidityGauge: LiquidityGauge
 }
 
 type Pool @entity {
+  " Pool contract address "
   id: ID!
+
+  " Pool ID on Balancer's Vault - nullable since it can be any contract "
   poolId: Bytes
+
+  " Pool contract address "
   address: Bytes!
+
+  " Most recent, unkilled gauge in the GaugeController "
   preferentialGauge: LiquidityGauge
+
+  " List of gauges created for the pool "
   gauges: [LiquidityGauge!] @derivedFrom(field: "pool")
 }
 
 type RewardToken @entity {
+  " Equal to: <tokenAddress>-<gaugeAddress> "
   id: ID!
+
+  " ERC20 token symbol - empty string if call reverts "
   symbol: String!
+
+  " ERC20 token decimals - zero if call reverts "
   decimals: Int!
+
+  " Reference to LiquidityGauge entity "
   gauge: LiquidityGauge!
+
+  " Amount of reward tokens that has been deposited into the gauge "
   totalDeposited: BigDecimal!
 }
 
 type GaugeShare @entity {
+  " Equal to: <userAddress>-<gaugeAddress> "
   id: ID!
+
+  " Reference to User entity "
   user: User!
+
+  " Reference to LiquidityGauge entity "
   gauge: LiquidityGauge!
+
+  " Balance of BPTs user has staked in the LiquidityGauge "
```

Comment:
While it's true that for the existing gauges this is equal to the amount of BPT the user has staked, in reality this is 
  " The user's balance of gauge deposit tokens "